tests/execution_test.go

```diff
@@ -209,6 +217,26 @@ func TestListWorkflowExecutions(t *testing.T) {
 	assert.Equal(t, len(resp.Executions), 4)
 }
 
+func TestListWorkflowExecutionsWithTags(t *testing.T) {
+	truncateAllTablesForTestingOnly()
```

Comment:
should we update [this](https://github.com/flyteorg/flyteadmin/blob/master/tests/bootstrap.go#L105) function to also truncate the newly added tables?